charts/das/templates/statefulset.yaml

```diff
@@ -34,6 +34,24 @@ spec:
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
       initContainers:
+      {{- if index .Values "configmap" "data" "data-availability" "rest-aggregator" "sync-to-storage" "state-dir"}}
+      - name: create-syncstate-directory
+        command:
+          - sh
+          - -c
+          - |
+            mkdir -p {{ index .Values "configmap" "data" "data-availability" "rest-aggregator" "sync-to-storage" "state-dir" }}
+            chown -R 1000:1000 {{ index .Values "configmap" "data" "data-availability" "rest-aggregator" "sync-to-storage" "state-dir" }}
```

Comment:
Is this chown necessary? I would think the default podsecuritycontext takes care of this.